workflow-service/src/main/java/com/redhat/parodos/security/SecurityUtils.java

```diff
@@ -48,4 +48,44 @@ public static String getUsername() {
 		return null;
 	}
 
+	public static String getEmail() {
```

Comment:
All the methods below use the same code as `getUsername`. Please fix

Reply:
yes, good catch! I will remove them